_overviews/reflection/overview.md

```diff
@@ -171,7 +171,7 @@ which makes all classes and types available that are loaded by the classloader
 that also loaded the class of `p` (`Purchase`), which we need in order to
 access member `shipped`.
 
-    scala> val shippingTermSymb = ru.typeOf[Purchase].declaration(ru.TermName("shipped")).asTerm
+    scala> val shippingTermSymb = ru.typeOf[Purchase].decls(ru.TermName("shipped")).asTerm
```

Comment:
The code as is (`ru.typeOf[Purchase].decls(ru.TermName("shipped")).asTerm`) produces the following error: "reflect.runtime.universe.MemberScope does not take parameters"

Reply:
fixed in #935?

Reply:
Yes